setup.py

```diff
@@ -53,7 +53,7 @@
 MAJOR               = 0
 MINOR               = 7
 MICRO               = 0
-SUFFIX              = ""  # Should be blank except for rc's, betas, etc.
```

Comment:
I think a development version is a pre alpha version. So we should also the version number. I did what I think it should be in this PR: https://github.com/Theano/Theano/pull/3861

So I'll close this PR. If you think we should not bump the version number at the same time. Tell us.

thanks for pushing this.